types/src/lib.rs

```diff
@@ -67,14 +67,15 @@ mod tagged;
 #[cfg(any(feature = "testing", test))]
 pub mod testing;
 mod timestamp;
-mod transaction;
+/// Module containing transaction types.
+pub mod transaction;
```

Comment:
why is this module made public? we re-export types from it below, we probably shouldn't make it public

Reply:
My bad good catch thanks 

https://github.com/casper-network/casper-node/pull/4714/commits/745b85cb4e42622d5b9b6f28abb7140f3211901e